pkg/dm/install.go

```diff
@@ -0,0 +1,184 @@
+package dm
+
+import (
+	"github.com/kubernetes/deployment-manager/pkg/format"
+	"github.com/kubernetes/deployment-manager/pkg/kubectl"
+)
+
+// Install uses kubectl to install the base DM.
+//
+// Returns the string output received from the operation, and an error if the
+// command failed.
+func Install(runner kubectl.Runner) (string, error) {
+	o, err := runner.Create([]byte(InstallYAML))
+	return string(o), err
+}
+
+// IsInstalled checks whether DM has been installed.
+func IsInstalled(runner kubectl.Runner) bool {
+	// Basically, we test "all-or-nothing" here: if this returns without error
+	// we know that we have both the namespace and the manager API server.
+	out, err := runner.GetByKind("rc", "manager-rc", "dm")
+	if err != nil {
+		format.Err("Installation not found: %s %s", out, err)
+		return false
+	}
+	return true
+}
+
+// InstallYAML is the installation YAML for DM.
+const InstallYAML = `
```

Comment:
This was added to the requirements so the user could download the binary and install server components with `helm dm install`.